rotkehlchen/data/importer.py

```diff
@@ -212,7 +214,11 @@ def import_cointracking_csv(
         with open(filepath, 'r', encoding='utf-8-sig') as csvfile:
             data = csv.reader(csvfile, delimiter=',', quotechar='"')
             header = remap_header(next(data))
+            imported_rows = 0
             for row in data:
+                imported_rows += 1
+                if imported_rows % ROWS_PER_YIELD == 0:
+                    gevent.sleep(0.5)
```

Comment:
you can context switch simply by `gevent.sleep(0)`. 0.5 may be too long even. Either use 0 or 0.1 as we use in the accounting.